package.json

```diff
@@ -8,10 +8,12 @@
     "clean": "rimraf dist",
     "start": "webpack-dev-server --hot --inline --progress",
     "build": "webpack --progress",
-    "prod": "npm run clean && webpack -p --progress"
+    "prod": "npm run clean && webpack -p --progress",
+    "winstop": "Taskkill /IM node.exe /F"
   },
   "dependencies": {
-    "bootstrap": "^3.3.7",
+    "ajv": "^6.9.2",
+    "bootstrap-vue": "^2.0.0-rc.13",
```

Comment:
We should consider updating package.json dependencies, as the ones inside are pretty old.